manifests/init.pp

```diff
@@ -70,22 +78,14 @@
   Boolean $manage_oddjobd = false,
   Variant[Boolean, Enum['running', 'stopped']] $service_ensure = 'running',
   Array $service_dependencies = [],
-  Array $enable_mkhomedir_flags = [
```

Comment:
leave these defaults alone, once you put it in hiera, that will work for your fedora versions

Reply:
Good idea!

Reply:
Done.